packages/mobile/src/account/saga.ts

```diff
@@ -106,7 +127,12 @@ export function* watchSetPincode() {
   yield takeLeading(Actions.SET_PINCODE, setPincode)
 }
 
+export function* watchClearStoredAccount() {
+  yield takeLeading(Actions.CLEAR_STORED_ACCOUNT, clearStoredAccountSaga)
```

Comment:
This can be just `take` because we can't have multiple dispatched per app session right?